src/tests/integration/gh878.at

```diff
@@ -0,0 +1,21 @@
+FWD_START_TEST([interfaces without connection profile])
+AT_KEYWORDS(gh878)
+
+NS_CHECK([ip link add dummy0 type dummy])
+
+START_NETWORKMANAGER()
+
+NS_CHECK([ip link show], 0, [ignore])
+echo NS_CMD([nmcli connection delete dummy0]) >> ./cleanup
+NS_CHECK([nmcli connection add type dummy ifname dummy0 con-name dummy0 ipv6.method ignore], 0, [ignore])
+NS_CHECK([nmcli connection up dummy0], 0, [ignore])
```

Comment:
[CI fails](https://github.com/firewalld/firewalld/actions/runs/4001478444/jobs/6867744159#step:15:1259) here trying to get an IP address. Need to disable ipv4.